spark/src/test/java/org/apache/iceberg/spark/source/TestIdentityPartitionData.java

```diff
@@ -110,24 +113,52 @@ public static void stopSpark() {
   private Table table = null;
   private Dataset<Row> logs = null;
 
-  @Before
-  public void setupTable() throws Exception {
+  /*
+  Use the Hive Based table to make Identity Partition Columns with no duplication of the data in the underlying
+  parquet files. This makes sure that if the identity mapping fails, the test will also fail.
```

Comment:
Nit: Looks like this Javadoc is missing the normal `* ` at the start of these lines?

Reply:
I was just leaving a multiline comment, but I'll make it a javadoc